doc/stdlib/arrays.rst

```diff
@@ -1045,7 +1051,7 @@ dense counterparts. The following functions are specific to sparse arrays.
    For an in-place version and algorithmic information, see :func:`Base.SparseArrays.dropzeros!`\ .
 
 .. function:: permute{Tv,Ti,Tp<:Integer,Tq<:Integer}(A::SparseMatrixCSC{Tv,Ti}, p::AbstractVector{Tp},
-..............    q::AbstractVector{Tq})
+                  q::AbstractVector{Tq})
```

Comment:
The dots aren't needed and mess up the `HTML` rendering (see end of http://docs.julialang.org/en/latest/stdlib/arrays/), though `genstdlib` will match them with or without.